examples/basic/terraform.tfvars

```diff
@@ -0,0 +1,11 @@
+namespace = "eg"
```

Comment:
rename the file from `terraform.tfvars` to `terraform.tfvars.example`

Reply:
done